src/hugr/views/sibling.rs

```diff
@@ -0,0 +1,572 @@
+//! Views for HUGR sibling subgraphs.
+//!
+//! Views into subgraphs of HUGRs within a single level of the
+//! hierarchy, i.e. within a sibling graph. Such a subgraph is
+//! represented by a parent node, of which all nodes in the sibling subgraph are
+//! children, as well as a set of edges forming the subgraph boundary.
+//! The boundary must be fully contained within the sibling graph of the parent.
+//!
+//! Sibling subgraphs complement [`super::HierarchyView`]s in the sense that the
+//! latter provide views for subgraphs defined by hierarchical relationships,
+//! while the former provide views for subgraphs within a single level of the
+//! hierarchy.
+//!
+
+// TODO:
+// //! This module exposes the [`SiblingView`] trait, which is currently
+// //! implemented by the [`SiblingSubgraph`] struct in this module, as well as
+// //! the [`SiblingGraph`] hierarchical view.
```

Comment:
Why the TODO? What'd be the goal of the trait?

(If not needed, remove the commented-out stub below too)

Reply:
I've removed the TODO. @doug-q had a PR for this #355 so I will leave it up to him to see if he wants to implement this.